trio/_highlevel_open_tcp_stream.py

```diff
@@ -126,7 +130,17 @@ def close_all():
             raise MultiError(errs)
 
 
-def reorder_for_rfc_6555_section_5_4(targets):
+def reorder_for_rfc_6555_section_5_4(
+    targets: list[
+        tuple[
+            AddressFamily,
+            SocketKind,
+            int,
+            str,
+            tuple[str, int] | tuple[str, int, int, int],
```

Comment:
`Address`

Reply:
See previous comment on `attempt_connect`

Reply:
Changed in e82ce69

Reply:
I think the error on this one could be ignored / worked around as it's just about invariance of List, but given that it's only used internally there's no reason to.